magento/magento2
# Issue 8258

Form validation with identical field names does not work as expected

### Description
Assign indices to array inputs and later remove them before submitting as a workaround for the validation library to work properly.

### Fixed Issues (if relevant)
1. magento/magento2#8258 Form validation with identical field names does not work as expected

### Manual testing scenarios
1. Create a form with identical fields and add JavaScript validation
2. Submit the form with improper values in fields.
3. Validation should work properly now
